scripts/json2wal/main.go

```diff
@@ -45,7 +45,7 @@ func main() {
 	}
 	defer walFile.Close()
 
-	br := bufio.NewReader(f)
+	br := bufio.NewReaderSize(f, 70000)
```

Comment:
Can you please add a comment here explaining why this was necessary. Will help anyone who looks at it in the future :) 

Would be even better if we could make it directly a function of the relevant variable - maybe `2 * types.BlockPartSizeBytes`  for safe keeping :)

Reply:
sure, I should have done it. :)